lease/lessor.go

```diff
@@ -173,11 +168,17 @@ func (le *lessor) SetRangeDeleter(rd RangeDeleter) {
 // TODO: when lessor is under high load, it should give out lease
 // with longer TTL to reduce renew load.
 func (le *lessor) Grant(ttl int64) *Lease {
-	id := LeaseID(le.idgen.Next())
-
 	le.mu.Lock()
 	defer le.mu.Unlock()
 
+	// granted leases numbers are consistent via raft-pass through
+	// TODO: this bakes in the lease assignment values; upgrading
```

Comment:
OK, I agree sending the lease id through raft is better; keeping the next lease id in sync is too delicate. Will change.